airbyte-integrations/connectors/destination-bigquery-denormalized/Dockerfile

```diff
@@ -4,7 +4,9 @@ WORKDIR /airbyte
 
 ENV APPLICATION destination-bigquery-denormalized
 
-ADD build/distributions/${APPLICATION}*.tar /airbyte
+COPY build/distributions/${APPLICATION}*.tar ${APPLICATION}.tar
```

Comment:
Please correct the file with master.

Reply:
More info: https://github.com/airbytehq/airbyte/pull/8516
We replace COPY in favor of ADD to reduce docker container size.

Reply:
This change ruins integration tests.
https://github.com/airbytehq/airbyte/runs/4570205306?check_suite_focus=true
I removed it back.